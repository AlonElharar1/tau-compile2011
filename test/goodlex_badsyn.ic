class Good {
	 int bad==4;
}