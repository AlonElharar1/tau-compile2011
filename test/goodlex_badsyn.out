1: CLASS
1: CLASS_ID(Good)
1: LCBR
2: INT
2: ID(bad)
2: EQUAL
2: INTEGER(4)
2: SEMI
3: RCBR
3: EOF
